classes/Services/LogsService.php

```diff
@@ -115,4 +116,19 @@ private function getDownloadLogsLabel(string $taskType): string
                 return $this->translator->trans('Download update logs');
         }
     }
+
+    /**
+     * @param TaskType::TASK_TYPE_* $taskType
+     */
+    private function getDownloadLogsTrackingEvent(string $taskType): string
+    {
+        switch ($taskType) {
+            case TaskType::TASK_TYPE_BACKUP:
+                return 'Backup logs downloaded';
+            case TaskType::TASK_TYPE_RESTORE:
+                return 'Restore logs downloaded';
+            case TaskType::TASK_TYPE_UPDATE:
+                return 'Update logs downloaded';
```

Comment:
Can we define constants for these events name? Likely in a new class `Events.php`